dashboards-observability/public/components/integrations/plugins/nginx/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { EuiBreadcrumb } from '@elastic/eui';
+import React from 'react';
+import { CoreStart } from '../../../../../../../src/core/public';
+import DSLService from '../../../../services/requests/dsl';
+import PPLService from '../../../../services/requests/ppl';
+import { Tabs } from '../../common/tabs';
+
+interface CustomPanelViewProps {
+  panelId: string;
+  page: 'app' | 'operationalPanels';
+  http: CoreStart['http'];
+  pplService: PPLService;
+  dslService: DSLService;
+  chrome: CoreStart['chrome'];
+  parentBreadcrumbs: EuiBreadcrumb[];
+  setToast: (
+    title: string,
+    color?: string,
+    text?: React.ReactChild | undefined,
+    side?: string | undefined
+  ) => void;
+  onEditClick: (savedVisualizationId: string) => any;
+  startTime: string;
+  endTime: string;
+  setStartTime: any;
+  setEndTime: any;
+  childBreadcrumbs?: EuiBreadcrumb[];
+  appId?: string;
+  updateAvailabilityVizId?: any;
+  onAddClick?: any;
+  tabId?: string;
+  appType: string;
+}
+
+export const NginxTab = (props: CustomPanelViewProps) => {

```

Comment:
If you make it export default, you do not have to import with specific name in the common file, hence you can make that code more generic.